src/js/chapterDOM.js

```diff
@@ -1,14 +1,31 @@
 import { makeDraggable, makeDroppable } from "./drag-drop";
 
 const createChapterStructure = (chapterObj, callback) => {
+  /* This method will show images and question and hide chapter story when player
+  clicks next btn on chapter story page */
+  const hideStoryConent = () => {
```

Comment:
I think you meant hideStoryContent, no?

Reply:
Yeah, wrong comment placement.

Reply:
Haha, now i realized that there is a typo. Definitely it should be content instead of conent :D